test/integration/api/admin/api_spec.js

```diff
@@ -17,6 +17,8 @@ const retry = require('../../../testUtils/helpers/retry');
 const {shouldTestFeature} = require("../../../spechelper");
 const API_V2 = cloudinary.v2.api;
 const DYNAMIC_FOLDERS = helper.DYNAMIC_FOLDERS;
+const assert = require('assert');
```

Comment:
`require` is the same as `import` to me and usually those go on the top of the file. Not a strong opinion though because `require` is written with variable assignment anyway.